test_conformance/extensions/cl_khr_external_semaphore/test_external_semaphore.cpp

```diff
@@ -51,9 +52,11 @@
         }                                                                      \
     } while (false)
 
-static const char* source = "__kernel void empty() {}";
+namespace {
```

Comment:
This isn't a hill I'm going to die on, but I have a slight preference for "static" vs. the anonymous namespace, if for no other reason than to reduce diffs.

Reply:
Corrected